src/rollup.ts

```diff
@@ -36,16 +37,33 @@ function rewriteInputsNamespace(code: string) {
 export async function bundleStyles({
   minify = false,
   path,
-  theme
+  theme,
+  files,
+  aliases
 }: {
   minify?: boolean;
   path?: string;
   theme?: string[];
+  files?: Set<string>;
+  aliases?: Map<string, string>;
 }): Promise<string> {
+  const assets = {
+    name: "resolve CSS assets",
+    setup(build: PluginBuild) {
+      build.onResolve({filter: /^\w+:\/\//}, (args) => ({path: args.path, external: true}));
+      build.onResolve({filter: /./}, (args) => {
+        if (args.path.endsWith(".css") || args.path.match(/^[#.]/)) return;
+        if (files) files.add(args.path); // /!\ modifies files as a side effect
```

Comment:
Modifying passed-in arguments again. It would be better to return another result with the assets.